tests/e2e/manifests/chi/test-037-1-storagemanagement-switch.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: "test-037-storagemanagement-switch"
+spec:
+  useTemplates:
+    - name: clickhouse-version
+  configuration:
+    zookeeper:
+      nodes:
+        - host: zookeeper
+          port: 2181
+    clusters:
+    - name: simple
+      layout:
+        shardsCount: 1
+        replicasCount: 2
+  defaults:
+#    storageManagement:
+#      provisioner: StatefulSet
+    templates:
+      podTemplate: multi-volume
+  templates:
+    volumeClaimTemplates:
+      - name: disk1
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+    podTemplates:
```

Comment:
If you are using clickhouse-version template in defaults above, you do not need podTemplate at all, it is automatically created from template